data/campaigns/The_Hammer_of_Thursagan/units/Dwarvish_Masked_Lord.cfg

```diff
@@ -14,7 +14,7 @@
     {AMLA_DEFAULT}
     cost=54
     usage=fighter
-    description= _ "Feared in dwarven legend is the Dwarf Lord who masks his face, using the formidable fighting powers of his kind without accounting to his peers and the loremasters of Dwarven-kind for the dark and bloody uses to which he puts his weapons. Like his barefaced kin, the Dwarf Lord wields axe and hammer with masterful skill, and can hit a target with a thrown hand axe at several paces. Though slow on their feet, these dwarves are a testament to the prowess of their kind."
+    description= _ "Feared in dwarven legend is the Dwarf Lord who masks his face, using the formidable fighting powers of his kind without accounting to his peers and to the loremasters of Dwarven-kind for the dark and bloody uses to which he puts his weapons. Like his barefaced kin, the Dwarf Lord wields axe and hammer with masterful skill, and can hit a target with a thrown hand axe at several paces. Though slow on their feet, these dwarves are a testament to the prowess of their kind."
```

Comment:
I don't think this extra "to" is actually necessary, though I guess it doesn't hurt.

Reply:
Without the "to" I would relate the "and" to "Feared" instead of "accounting"